packages/protocol/contracts/governance/Election.sol

```diff
@@ -817,9 +817,22 @@ contract Election is
    * @notice Returns a list of elected validators with seats allocated to groups via the D'Hondt
    *   method.
    * @return The list of elected validators.
-   * @dev See https://en.wikipedia.org/wiki/D%27Hondt_method#Allocation for more information.
    */
   function electValidatorSigners() external view returns (address[] memory) {
+    return electValidatorSigners(electableValidators.min, electableValidators.max);
+  }
+
+  /**
+   * @notice Returns a list of elected validators with seats allocated to groups via the D'Hondt
+   *   method.
+   * @return The list of elected validators.
+   * @dev See https://en.wikipedia.org/wiki/D%27Hondt_method#Allocation for more information.
+   */
+  function electValidatorSigners(uint256 minElectableValidators, uint256 maxElectableValidators)
+    external
```

Comment:
Wouldn't this function need to be `public` now since it's called from within the contract?

Reply:
Definitely